src/usr/bin/google_set_multiqueue

```diff
@@ -213,12 +266,21 @@ num_cpus=$(nproc)
 
 num_queues=0
 for q in $XPS; do
+  interface=$(echo "$q" | grep -oP 'net/\K[^/]+')
+  if [[ $IS_A3_PLATFORM ]] && ! $(is_gvnic "$interface"); then
```

Comment:
Sorry, missed one more thing here. This needs to be `[[ $IS_A3_PLATFORM == 0 ]]`. Current syntax will evaluate to true as long as the length of IS_A3_PLATFORM is non-zero which is true both when it's on A3 and when it's not.

Reply:
No problem, thanks for spot checking as I'm not that familiar with bash script syntax. :)